maestro-client/src/test/java/maestro/UiElementTest.kt

```diff
@@ -0,0 +1,72 @@
+package maestro
+
+import org.junit.jupiter.api.Test
+import com.google.common.truth.Truth.assertThat
+
+internal class UiElementTest {
```

Comment:
Could you also add a case to `IntegrationTests` with an `assertVisible` command?

Reply:
Done